### DISCUSSION TITLE
COM Port connection

### DISCUSSION THREAD

**Discussion by astroalex80** [mworion/MountWizzard4#359] [Feature Request] [answered]
Hi Marion,

is there any chance a connection via COM Port will be included in near future?

Thanks,
Alex

**Comment by mworion** [comment, marked as answer]
Hi Alex,
I apologize, but a connection via COM Port ist not feasible as I use multiple connections in the same time a this is not supported on COM port. In addition I will use in the next version more Web Interfaces provided by the mount (Upload). This is also not possible via COM.
Michel
